docs/DeveloperGuide.md

```diff
@@ -354,6 +354,8 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | user    | specify if an amount of money in the debt is inclusive or exclusive of GST and service charge | I do not have to manually calculate the final debt amount           |
 | `* *`    | user    | see an overview of all the debts owed                                                         | I am in better control of my overall financial situation            |
 | `* *`    | user    | search for a person’s contact                                                                 | I can easily access his contact details                             |
+| `* *`    | user    | search for people who owe me money for a certain event                                        | I can look for them to recover my money                             |
+| `* *`    | user    | search for people who owe me money past a certain date                                        | I can ensure that these people do not forget to return my money     |
```

Comment:
Maybe add "for too long" after the end of the `so that...` to make it less generic and more relevant to searching by date.
IDK how to make the event `so that...` less generic though